src/index.js

```diff
@@ -345,36 +345,47 @@ export class ULabel {
         ul.resize_observers.push(tb_overflow_resize_observer)
 
         // Listener for soft id toolbox buttons
-        $(document).on("click.ulabel", "#" + ul.config["toolbox_id"] + ' a.tbid-opt', (e) => {
+        $(document).on("click.ulabel", "#" + ul.config["toolbox_id"] + " a.tbid-opt", (e) => {
             let tgt_jq = $(e.currentTarget);
             let pfx = "div#tb-id-app--" + ul.state["current_subtask"];
             const subtask_key = ul.state["current_subtask"];
             const current_subtask = ul.subtasks[subtask_key];
             if (tgt_jq.attr("href") === "#") {
-                $(pfx + " a.tbid-opt.sel").attr("href", "#");
-                $(pfx + " a.tbid-opt.sel").removeClass("sel");
+                let current_id_button = $(pfx + " a.tbid-opt.sel");
+                current_id_button.attr("href", "#");
+                current_id_button.removeClass("sel");
+                let old_id = parseInt(current_id_button.attr("id").split("_").at(-1));
```

Comment:
```suggestion
                const current_id_button = $(pfx + " a.tbid-opt.sel");
                current_id_button.attr("href", "#");
                current_id_button.removeClass("sel");
                const old_id = parseInt(current_id_button.attr("id").split("_").at(-1));
```
Use `const` for pointers and variables that don't change in value.